### DISCUSSION TITLE
Installing the correct duckdb version

### DISCUSSION THREAD

**Discussion by LarchLiu** [dcaribou/transfermarkt-datasets#227] [FAQ] [answered]
I installed `duckdb` with brew, version 0.9.0. But when I query sql with duckdb I get the error `Error: Trying to read a database file with version number 51, but we can only read version 64. version 64.`. It looks like dbt is using an older version 0.8.1 of duckdb. I want to know how to  upgrade dbt's duckdb version to 0.9.0.

**Comment by dcaribou** [comment, marked as answer]
The `dbt-duckdb` 1.6.0 package (the latest version available as of today) comes with an embedded duckdb installation on the 0.8.x version. We cannot update that yet to 0.9.x until a new dbt-duckdb version is released (they usually take a few weeks since to upgrade to a new duckdb version).

What you can do, though, is to install the duckdb 0.8.x version. I did it like this
```console
wget https://github.com/duckdb/duckdb/releases/download/v0.8.1/duckdb_cli-osx-universal.zip
unzip duckdb_cli-osx-universal.zip
./duckdb --version
```
